website/templates/include/profile/schools.mako

```diff
@@ -29,9 +29,9 @@
 
                     <div class="form-group">
                         <label>Institution</label>
-                        <input class="form-control" data-bind="value: institution" 
```

Comment:
This line was un-indented a single space - it no longer matches the level above it.